content/01-install/01a-install-client-windows.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Install Client on Windows
+metaTitle: Install Client on Windows
+---
+>Author's note to reviewers: the current general purpose instructions do not suit Windows. If the client is installable on Windows, I need someone to give me instructions. At present, it doesn't look to me like there is a Windows installer on the website.
+
+Before you start, the team recommends that you read [Getting Started](/first-steps/getting-started) if you haven't already. It's a 30-second read that will save you time.
+
+Mantis is both a graphical wallet and a command line client. You can download either or both.
+
+This section describes how to install the Mantis client on your Windows machine. For instructions to install the Mantis wallet, you can go [here](/install/-install-wallet-windows).
+
+
+## Prerequisites
+
+These are the prerequisites for installing Mantis client:
+* A Java Virtual Machine (JVM) with version 1.8.x.
+    To check the JVM version, use
+     `java -version`.
+    To install Java, follow [these instructions](https://java.com/en/download/help/download_options.html).
+    (Note that the Mantis team has not tested the client with JVM 1.9 or later versions)
+* 4G of memory (RAM)
+    Additional RAM is needed for the DAG file _if mining_ is enabled.
+* 250GB of disk space to ensure fast sync of the node. You will need more than that for the future growth of the chain. An SSD of at least 500G is recommended for full sync.
+
+
+## Installing Mantis client
+
+To install the Mantis MantisClient, follow these steps:
+
+1. Download the client binary from the 
+
+2. Choose the Windows 64 bit installer to download from the [Mantis Client page](https://mantisclient.io/):
+![Mantis Client page](../images/1-Download-Client.png)
+
+3. Note the checksum.
+> Remember to run checksum verification on your downloads. Refer to [this section](/how-tos/how-check-hash-windows) for instructions.
+
+```
+mv mantis-3.1.0.zip /home/<user>/bin/
+cd /home/<user>/bin/
+unzip mantis-3.1.0.zip
+rm mantis-3.1.0.zip
+```
+
+3. (Optional) Rename your mantis bin folder for ease of future updates.
+
+```
+mv /home/<user>/bin/mantis-3.1.0 /home/<user>/bin/mantis
+```
+
+> Default folders for applications on Linux are `/home/<user>/Applications` and `/home/<user>/bin`, or `/usr/local/bin`. We recommend placing the mantis folder in there.
```

Comment:
I would honestly recommend either `/home/<user/mantis` (or more flexible: `~/mantis` where `~` is the shorthand for the current users home folder) or nothing at all. Linux users will generally have an idea about this themselves.